### PR TITLE
Prepare first release

### PR DESCRIPTION
**Pull request by ldko**
This is to prepare for the first release. ping @somexpert @madhulika95b

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,16 +4,23 @@ is being developed according to the [ait-specification](https://github.com/WASAP
 
 ## Requirements
 
-* Python 3.4, 3.5, or 3.6
+* Python 3.4-3.7
 
 ## Installation
 
-The WASAPI client may be installed with:
+To run the latest code, the WASAPI client may be installed with:
```
Comment (somexpert) [inline README.md]:
```suggestion
To run the latest code, the WASAPI client may be downloaded from GitHub and installed with:
```

File: README.md [hunk 2]
```diff
@@ -4,16 +4,23 @@ is being developed according to the [ait-specification](https://github.com/WASAP
...
 
 ```
  $ python setup.py install
```
Comment (somexpert) [inline README.md:16]:
```suggestion
 $ git clone https://github.com/unt-libraries/py-wasapi-client.git
 $ cd py-wasapi-client
 $ python setup.py install
```

File: README.md [hunk 3]
```diff
@@ -4,16 +4,23 @@ is being developed according to the [ait-specification](https://github.com/WASAP
...
 ```
 
+Alternatively, the most recent release (not guaranteed to be the latest
+code) may be installed from [PyPi](https://pypi.org/):
```
Comment (somexpert) [inline README.md]: Could link directly to the upcoming py-wasapi-client page:
```suggestion
code) may be installed from [PyPi](https://pypi.org/project/py-wasapi-client/):
```

**Review by somexpert — APPROVED** [review]
Couple small optional suggestions, but nothing blocking. :thumbsup:

**Review by madhuBayy — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ldko) [reply README.md:16]: Does the new wording work for you?

Comment (somexpert) [reply README.md:16]: yep!

**Pull request merged**